src/EtherFiNodesManager.sol

```diff
@@ -68,6 +68,9 @@ contract EtherFiNodesManager is
 
     mapping(address => bool) public eigenLayerOperatingAdmin;
 
+    mapping(address => mapping(bytes4 => bool)) public allowedOperatorCalls;
```

Comment:
i believe it was for a separate proposal?

Reply:
combined with this update to ensure it actually met the functional requirements of its first intended application